### DISCUSSION TITLE
This thing is useless, want it off

### DISCUSSION THREAD

**Discussion by wizzomafizzo** [community/community#37722] [Other Feature Feedback, Questions, & Ideas]
Don't care at all what random people I've never met are doing and the explore page seemed fine for that anyway. I'm not just having to sift through extra information, this silly feed thing is actively hiding important activity in my repos and what I've *actually* subscribed to.

Keep pushing it if you like but there needs to be an option to disable it.

**Comment by ossanna16** [comment]
Hi there @wizzomafizzo and welcome to our community! Thank you for the feedback!
